### PR TITLE
[chore] move heimdall-fs-monitor to monorepo

### PR DESCRIPTION
**Pull request by gabrielcsapo**
Apart of #109

### CODE REVIEW

**Review by hjdivad — APPROVED** [review]

**Review by hjdivad — COMMENTED** [review]
@gabrielcsapo what are the other fixes you're referring to?

**Comment by hjdivad** [comment]
will wait for CI but broadly LGTM

@gabrielcsapo given that we're just re-orging to monorepo I presume you don't need a release cut.  Is that correct?

**Comment by gabrielcsapo** [comment]
@hjdivad sorry copied and pasted the old prs status line.

Correct we don't need a release cut. Once we finish up #109 we can cut a new release for all the modules with their new location.

**Pull request merged**
